generate-audio.py

```diff
@@ -61,6 +67,33 @@ def load_captions(config):
         raise Exception('Unsupported subtitles format')
 
 
+# TODO figure out better way of defining break length
+def define_break(diff_length, num_of_pauses):
+    length_of_pause = diff_length / num_of_pauses
+    if diff_length / num_of_pauses > 2:
+        return 1000
+    elif 1 < length_of_pause < 2:
+        return 800
+    else:
+        return 500
+
+
+def extend_sentence_audio(sentence_audio, caption):
+    audio_duration = sentence_audio.duration_seconds
+    caption_start_time = caption_start(caption)
+    caption_end_time = caption_end(caption)
+    diff = ((caption_end_time - caption_start_time) - audio_duration).__round__(3)
+    result = ''
+    split_caption = caption.text.split(',')
+    if len(split_caption) == 1:
+        return sentence_audio
+    for idx, cpt in enumerate(split_caption):
+        result = result + cpt
+        if idx != len(split_caption):
+            result = result + '<break time="{}ms"/>'.format(define_break(diff, len(split_caption) - 1))
```

Comment:
Jeśli to nie jest ostatnia część "captiona" doklejamy tag ciszy o konkretnej długości obliczonej przez `define_break`